assets/styles/style.css

```diff
@@ -0,0 +1,282 @@
+* {
+  box-sizing: border-box;
+  overflow-x: hidden;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+/* Home */
+
+.home-container {
+  background-image: url("../images/wedding_couple.jpg");
+  background-position: center;
+  background-size: cover;
+  min-height: 600px;
+  filter: grayscale(50%);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.home-container h1,
+.home-p2,
+.home-p3 {
+  color: white;
+  text-align: center;
+}
+
+.home-container h1 {
+  font-size: 70px;
+}
+
+.home-p2,
+.home-p3 {
+  font-size: 25px;
+  margin-top: -15px;
+}
+
+.home-p3 {
+  margin-top: -10px;
+  font-weight: bold;
+  font-style: italic;
+}
+
+/* Jane */
+
+.jane-container1 {
+  filter: grayscale(50%);
+  background-color: #fdd;
+  padding-bottom: 80px;
+}
+
+.jane-container2 {
+  max-width: 900px;
+  margin: auto;
+}
+
+.jane-container2 h3 {
+  text-align: center;
+  margin-top: 80px;
+  color: gray;
+  font-size: 35px;
+}
+
+.jane-container2 img {
+  filter: grayscale(50%);
+}
+
+.jane-container2 p {
+  margin-top: 40px;
+  font-style: italic;
+  line-height: 30px;
+}
+
+.jane-btn {
+  margin-left: 40%;
+  margin-top: 30px;
+  border: none;
+  padding: 15px;
+  font-size: 18px;
+  border-radius: 5px;
+  color: white;
+  background-color: black;
+}
+
+.jane-btn:hover {
+  color: black;
+  background-color: #ccc;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] mobile first first design. expected min-width definition
- [ ] I expect just one definition. as design is same tablet and mobile